tensorboard/webapp/app_routing/types.ts

```diff
@@ -40,6 +40,16 @@ export enum RouteKind {
 
 export const DEFAULT_EXPERIMENT_ID = 'defaultExperimentId';
 
+/**
+ * `declare` to express this as a public API that is not to be mangleable.
+ * Because `experimentIds` are declared in routes as string literals, we cannot
+ * mangle the property name.
+ */
+export declare interface CardRouteParams {
+  experimentIds: string;
+  tag: string;
```

Comment:
I don't get why this interface is needed, will tag attribute be used later? (At least not in the code I can see)

Reply:
I created this interface to provide accurate context so that the route params for Card Route wouldn't be called CompareRouteParams, but right now there's not plan to use tag later. I can remove tag and just keep the CardRouteParams, or I can use the CompareRouteParams for the Card Route. Do you have a preference?